otftotfm/automatic.cc

```diff
@@ -313,7 +326,7 @@ update_odir(int o, String file, ErrorHandler *errh)
     String ls_r = writable_texdir + "ls-R";
     bool success = false;
     if (access(ls_r.c_str(), R_OK) >= 0) // make sure it already exists
-        if (FILE *f = fopen(ls_r.c_str(), "a")) {
+        if (FILE *f = fopen(ls_r.c_str(), "ab")) {
```

Comment:
Why are all files being made binary? Is there an actual concrete reason? Because this is just a text file. But maybe on Windows all kpathsea files such as `ls-R` should use Unix lineendings.